flake.nix

```diff
@@ -25,9 +25,9 @@
     };
   };
 
-  outputs = { self, nixpkgs, flake-parts, ... }:
+  outputs = inputs@{ self, nixpkgs, flake-parts, ... }:
```

Comment:
Can you add a comment referencing the issue? Might be useful for future nix people

Reply:
you automatically get a message in your terminal when you run `nix flake show` when you do it the old way, it's an api change for [flake-parts](hercules-ci/flake-parts) and all new flake parts projects do it correclty by default, so I don't see this being neccesary